### ISSUE TITLE
Length of extremity

### ISSUE THREAD

**Issue by JakobThumm** [Sven-Schepp/SaRA#2]
Hi Sven,

if I recall correctly, we define extremities as entire arms / legs.
Therefore, the extremity should start at e.g. the shoulder joint and have the max length of the arm.

However, in Articulated_Pos we init the extremity with
https://github.com/Sven-Schepp/SaRA/blob/97c059b82297c67a1831fcbe43bc568e5a2f8ec1/ReachLib/src/articulated_pos.cpp#L43

I thought, that thickness should be the value of the hand / foot and length should be the length of all links connecting a shoulder to the hand.

I believe that we either need to redefine the constructor here or make it very clear what values thickness and length should be!

**Comment by Sven-Schepp** [comment]
The assesment is correct and warrants according action. Both options seem fine. Additionally, we could require the user to measure thickness+length (instead of both separately) for G_POS and only pass that along to the constructor.

**Comment by JakobThumm** [comment]
There is also the possibility to leave out the length and define the joints that define an extremity. 
Thereby, we measure the distance from joint to joint in each timestep and sum up the lengths to get the extremity length.
E.g. length(arm) = length(|pos(shoulder) - pos(elbow)|) + length(|pos(elbow) - pos(wrist)|) + thickness(hand)
This is more flexible but would need some code adjustments.

**Comment by Sven-Schepp** [comment]
This would be a legacy approach that we have dropped in favor of constant initially measured lengths. We have found that by constantly calculating the distance between joints there is an increased dependence on the accuracy of the mocap system. This leads to unnatural 'wobbling' of G_POS dimensions when using our system and in CMU's recordings.  This approach is fine, if we can provably avoid this effect.
